src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -335,4 +352,28 @@ private function configureStorage(array $config, ContainerBuilder $container, Lo
 
         $container->setAlias('sulu_media.storage', 'sulu_media.storage.' . $storage)->setPublic(true);
     }
+
+    private function getSupportedMimeTypes($ghostScriptPath, $ffmpegBinary, $ffprobeBinary)
+    {
+        $mimeTypes = ['image/*'];
+
+        if ($ffmpegBinary
+            && $ffprobeBinary
+            && $this->checkCommandAvailability($ffmpegBinary)
+            && $this->checkCommandAvailability($ffprobeBinary)
+        ) {
+            $mimeTypes[] = 'video/*';
+        }
+
+        if ($ghostScriptPath && $this->checkCommandAvailability($ghostScriptPath)) {
+            $mimeTypes[] = 'application/pdf';
+        }
+
+        return $mimeTypes;
+    }
+
+    private function checkCommandAvailability($command)
+    {
+        return null !== $this->executableFinder->find($command);
```

Comment:
@danrot maybe we should go with something like because the executeableFinder search only in PATHs dirs we would need to at the given dir as extraDir if its not inside the PATH:

```
return null !== $this->executableFinder->find(basename($command), null, [dirname($command)]);
```